serversideapi/src/main/java/com/example/serversideapi/Greeting.java

```diff
@@ -1,5 +1,6 @@
 package com.example.serversideapi;
 
+// Create structure of api
```

Comment:
Kidding Me? Of course it's a POJO and doesn't need additional comment

Reply:
Okay:)
I'll change it